Autopilot/AttitudeManager/SensorFusion.hpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * Sensor Fusion - Converts raw sensor data into human readable formatted structs
+ * Author: Lucy Gong
+ */
+#ifndef SENSORFUSION_HPP
+#define SENSORFUSION_HPP
+
+//Include involved sensors
+#include "IMU.hpp"
+#include "airspeed.hpp"
+
+struct SFOutput_t{
+    float IMUroll, IMUpitch, IMUyaw; //in rad (for now)
+    float IMUrollrate, IMUpitchrate, IMUyawrate; //in rad/s (for now)
+
+    float Airspeed; //in m/s (for now)
+};
+
+struct SFError_t{
+    int errorCode;
+};
+
+/**
+ * Converts raw IMU output into roll, pitch and yaw angles
+ * 
+*/
+void IMUconv(IMUData_t *imu);
+
+
+SFError_t SF_GetResult(SFOutput_t *Output, SFError_t *Error);
```

Comment:
Also, why do we need 2 seperate functions to do this job ?

Reply:
The intention was to keep the IMU conversion separate so it would be easy to find visually in the file - SF_GetResult() calls IMUconv. Since I can achieve the same effect by both declaring and defining IMUconv in SensorFusion.c, I will move this fn there.